### PR TITLE
January 2021 newsletter draft

### PR DESCRIPTION
**Pull request by jvdp1**
@milancurcic @LKedward @gklimowicz @certik 

Here is a draft for the January 2021 newsletter.
I tried to update for fortran-lang.org and stdlib sections. Other sections (fpm, compilers, ...) must be still be updated.

### CODE REVIEW

**Comment by milancurcic** [comment]
Thanks a lot for starting it @jvdp1!

**Comment by awvwgk** [comment]
I'll have a look at the fpm part.

File: _posts/2021-01-01-Fortran-Newsletter-January-2021.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+Update compilers page following Intel oneAPI release
+
+Ongoing work:
+
+* [#160](https://github.com/fortran-lang/fortran-lang.org/pull/160) (WIP):
+In-depth introduction for Fortran with Make.
+* [#186](https://github.com/fortran-lang/fortran-lang.org/pull/186) (WIP):
+Add missing packages from the list of popular Fortran projects
+* [#187](https://github.com/fortran-lang/fortran-lang.org/pull/187) (WIP):
+Correct Compiler page and tutorial regarding Intel oneAPI and PGI to NVIDIA
```
Comment (LKedward) [inline _posts/2021-01-01-Fortran-Newsletter-January-2021.md:38]:
```suggestion
Correct compiler page and tutorial regarding Intel oneAPI and PGI to NVIDIA
```

File: _posts/2021-01-01-Fortran-Newsletter-January-2021.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+Please try it out and let us know how it works, if there are any issues, or if the API can be improved.
+
+## Fortran Package Manager
+
+Here's what's new in `fpm`:
+
+* [Alpha release version 0.1.3](https://github.com/fortran-lang/fpm/releases/tag/v0.1.3)
+* [setup-fpm action](https://github.com/marketplace/actions/setup-fpm):
+  GitHub Action to setup the Fortran Package Manager on Ubuntu, MacOS and Windows runners
+* [discussion board](https://github.com/fortran-lang/fpm/discussions):
```
Comment (LKedward) [inline _posts/2021-01-01-Fortran-Newsletter-January-2021.md:104]:
```suggestion
* [Discussion board](https://github.com/fortran-lang/fpm/discussions):
```

File: _posts/2021-01-01-Fortran-Newsletter-January-2021.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+* [#230](https://github.com/fortran-lang/fpm/pull/230),
+  [#261](https://github.com/fortran-lang/fpm/pull/261) (WIP):
+  Specification of the fpm CLI.
+* [#316](https://github.com/fortran-lang/fpm/pull/316) (WIP):
+  Update subcommand "new" to reflect the addition of support for examples
+
+`fpm` is still in early development and we need as much help as we can get.
+Here's how you can help today:
+
+* Use it and let us know what you think! Read the [fpm packaging guide](https://github.com/fortran-lang/fpm/blob/master/PACKAGING.md) to learn how to build your package with fpm, and the [manifest reference](https://github.com/fortran-lang/fpm/blob/master/manifest-reference.md) to learn what are all the things that you can specify in the fpm.toml file.
```
Comment (LKedward) [inline _posts/2021-01-01-Fortran-Newsletter-January-2021.md:174]:
```suggestion
* Use it and let us know what you think! Read the [fpm packaging guide](https://github.com/fortran-lang/fpm/blob/master/PACKAGING.md) to learn how to build your package with fpm, and the [manifest reference](https://github.com/fortran-lang/fpm/blob/master/manifest-reference.md) to learn what are all the things that you can specify in the fpm.toml file.
* Browse existing *fpm* packages on the [fortran-lang website](https://fortran-lang.org/packages/fpm)
```

**Comment by LKedward** [comment]
Thanks for starting @jvdp1 and thanks for populating the *fpm* section @awvwgk - it was a busy month for *fpm*!
Left a few minor suggestions but everything else looks good. I'll update the contributor data tomorrow.

File: _posts/2021-01-01-Fortran-Newsletter-January-2021.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+You can watch the recording below:
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/S_xQCSRlefE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+As usual, subscribe to the [mailing list](https://groups.io/g/fortran-lang) and/or
+join the [Discourse](https://fortran-lang.discourse.group) to stay tuned with the future meetings.
+
+* [Episode 6](https://adspthepodcast.com/2021/01/01/Episode-6.html) of the
+Algorithms+Data Structures=Programs (ADSP) Podcast discussed Fortran and
+the recent fortran-lang developments.
```
Comment (jvdp1) [inline _posts/2021-01-01-Fortran-Newsletter-January-2021.md:212]: Nice! I was not aware of this podcast! The community 's efforts seem to pay off.

File: _posts/2021-01-01-Fortran-Newsletter-January-2021.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+The short-term goal of fpm is to make development and installation of Fortran packages with dependencies easier.
+Its long term goal is to build a rich and decentralized ecosystem of Fortran packages and create a healthy
+environment in which new open source Fortran projects are created and published with ease.
+
+## Compilers
+
+### Classic Flang
+
+TO DO
```
Comment (gklimowicz) [inline _posts/2021-01-01-Fortran-Newsletter-January-2021.md:189]:
```
There are several pull requests out for evaluation.

Only one pull request was merged in December:

* [PR#951 Fix for ICE in atomic instruction generation](https://github.com/flang-compiler/flang/pull/951)
```

File: _posts/2021-01-01-Fortran-Newsletter-January-2021.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+
+The Classic Flang biweekly call has been set up to discuss issues and plans
+for the next pull requests to be validated and merged. Our next calls are Wednesday, December 16 and 30, 8:00 AM Pacific time. The notes from previous calls, upcoming agenda and a link to join the call can be found
+[here](https://docs.google.com/document/d/1-OuiKx4d7O6eLEJDBDKSRnSiUO2rgRR-c2Ga4AkrzOI).
+
+### LLVM Flang
+
+TO DO
```
Comment (gklimowicz) [inline _posts/2021-01-01-Fortran-Newsletter-January-2021.md:197]:
```
Alexis-Perry Holby (aperry@lanl.gov) has taken over the Flang biweekly calls.
An invitation was sent to the _flang-dev_ LLVM email list on December 22nd.
Call notes will be sent to the _flang-dev_ email list and also recorded [here]( https://docs.google.com/document/d/10T-S2J3GrahpG4Ooif93NSTz2zBW0MQc_RlwHi0-afY).

Recent development updates:

* Semantic analysis fix for index-names of `FORALL` statements
* Continued work on parser support for `ALLOCATE`
* Build tidying
* OpenMP semantic checks: atomic, flush
* Continued work on new driver
* Fix for list-directed REAL output editing
* Bug fixes: USE of USE of generic, crash in folding (#48437), crash in OpenMP semantic check (#48308), `IMPLICIT_NONE(EXTERNAL)`
* Implement `STORAGE_SIZE()`, `SIZEOF()`, and `C_SIZEOF()`
* OpenACC: update serial construct clauses for 3.1, enforce restriction on routine directive and clauses
* OpenMP: adding important clauses to OmpClause, task_reduction clause
```

File: _posts/2021-01-01-Fortran-Newsletter-January-2021.md (hunk 5 above)
Comment (gklimowicz) [reply _posts/2021-01-01-Fortran-Newsletter-January-2021.md:189]: I'm not sure how to add a specific commit to this PR. So here are one brief Flang updates in the comments.

I left NVIDIA last Wednesday, and am no longer directly involved in the Flang projects. I am hoping that either Alexis Perry or someone at NVIDIA will take on the monthly Flang update duties here.

(I don't have permissions to push a commit Jeremie's branch and I'm too dumb to know if there is another way to do it.)

**Comment by milancurcic** [comment]
Thank you, @gklimowicz!

@certik are there any updates for LFortran or is this good to go?

**Comment by certik** [comment]
There are updates, but I don't have time to write them down, so this is good to go.

**Comment by milancurcic** [comment]
Okay, sounds good, we can defer LFortran news to the next newsletter. I'll merge now. Thank you all!

**Pull request merged**